### DISCUSSION TITLE
Button icon colour change based on numeric state <=

### DISCUSSION THREAD

**Discussion by townsmcp** [Clooos/Bubble-Card#790] [Questions about config, custom styles and templates] [answered]
Hi,
I am looking for a bit of help. I have seen on the Templates section you can set an icon colour based on state:
```
styles: |
  .bubble-icon {
    color: ${hass.states['light.your_light'].state === 'on' ? 'green' : 'red'} !important;
  }
```
Is it possible to do the same thing for a numeric state? For example I have a water softener and a sensor that tells me the numeric state as a percentage of salt block left. so rather than a 1 to 1 state (10% left), I would like to colour the icon based on less than or equal to 10% left

**Comment by townsmcp** [comment]
I have found a halfway house so far, but not exactly what I’m looking for:
```
.bubble-icon {
color: red !important;
```

**Reply by davosje** [reply, marked as answer]
Asked chatgpt for you. You mean something like this?

```
.bubble-icon {
    color: ${hass.states['sensor.salt_level'].state <= 10 ? 'red' : 'green'} !important;
}
```

**Reply by townsmcp** [reply]
@davosje lol I didnt think about using ChatGPT to get an answer. Out of curiosity, what question did you send it?

**Reply by davosje** [reply]
Happy it worked! 

What I did was really easy, copied your question and pasted it in chatgpt ;) 

You can search for gpt's in the left corner and then i'm using 'HomeAssistant Helper'.
